ecosystem/sep-0011.md

```diff
@@ -170,13 +172,19 @@ A few aggregate values are special-cased:
 
 ##### Asset
 
-The `Asset` type is rendered as a string in one of two forms:
+The `Asset` type is rendered as a string in one of three forms:
   - `native` (or any string up to 12 characters not containing an unescaped colon) for the native asset.
-  - `Code:IssuerAccountID` for issued assets.
+  - `Code:IssuerAccountID` for issued assets, i.e. assets of type
+    `ASSET_TYPE_CREDIT_ALPHANUM4` and `ASSET_TYPE_CREDIT_ALPHANUM12`.
 
     "Code" must consist of printable ASCII characters (octets 0x21 through 0x7e).
 
     The sequence `\x` introduces a hex escape sequence, e.g., `\x00` to introduce a 0-valued byte.  Otherwise, `\` escapes the next character, so `\\` is required to introduce a backslash. Stellar disallows assets of type `ASSET_TYPE_CREDIT_ALPHANUM12` that have fewer than 5 bytes, but such assets can be represented in binary XDR, and so in txrep are rendered with trailing `\x00` (escaped NUL bytes) to as to make the length 5---e.g., the 12-byte asset code `ABC` is rendered `ABC\x00\x00`. Note that stellar-core disallows non ASCII bytes in `AssetCode` fields, so the primary use of this feature is to construct or examine invalid transaction test cases.
+  - `LiquidityPoolID:LP` for liquidity pool shares, i.e. assets of type `ASSET_TYPE_POOL_SHARE`.
+
+    "LiquidityPoolID" is the hex encoded `PoolID`.
```

Comment:
Should we use `LP:$ID`, instead, to parallel the way `issuer` in `code:issuer` is the main differentiator / unique component?

Reply:
I also like the prefix format `LP:...`, but unfortunately there's an opportunity for ambiguity since `LP` could be a valid asset code. It makes it harder to programmatically determine if the text after the colon is an issuer account ID or a liquidity pool. This problem is exasperated if the network adds new strkey types that can be used as the issuer account ID and that overlap with the valid character sets liquidity pool IDs.